coldfront/plugins/qumulo/views/update_allocation_view.py

```diff
@@ -190,12 +190,24 @@ def _handle_attribute_change(
         attribute_name: str,
         form_value: Union[str, int],
     ) -> None:
-        attribute = AllocationAttribute.objects.get(
-            allocation_attribute_type=AllocationAttributeType.objects.get(
-                name=attribute_name
-            ),
-            allocation=allocation,
-        )
+        # some attributes are optional and so may not exist
+        try:
```

Comment:
@peaches7083 this will work; however, would `AllocationAttributeType.objects.get_or_create` be more appropriate?

Reply:
Something like this
```
alloacation_attribute = AllocationAttribute.objects.get_or_create(
        allocation_attribute_type=AllocationAttributeType.objects.get(
                name=attribute_name
            ),
            allocation=allocation,
        ) ,
        defaults={value=""}
)
```

Reply:
Hmm...yeah, that should work